### PR TITLE
DTSPO 9692 Initial structure of new centralised pipeline 

### PR DESCRIPTION
**Pull request by taherkhan30**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-9692

### Change description ###

Migrating current pipeline to use centralised templates  

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by timja — CHANGES_REQUESTED** [review]
labels should not have the location added in them it's all uksouth, 

seems to have happened in some unreviewed changes in https://github.com/hmcts/cnp-azuredevops-libraries

revert PR pending a proper fix:
https://github.com/hmcts/cnp-azuredevops-libraries/pull/91

**Comment by madjava** [comment]
@timja Tested out changed for a repo that those not need the mult iregion feature. Seems to work (i belive) pointing to PR at the mo but would be updated after the cnp bit is in master.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -11,163 +11,127 @@ resources:
   repositories:
   - repository: cnp-azuredevops-libraries
     type: github
-    ref: master
+    ref: DTSPO-9663-template-file-cleanup
```
Comment (timja) [inline azure-pipelines.yml:14]: please get this merged first

File: azure-pipelines.yml [hunk 2]
```diff
@@ -11,163 +11,127 @@ resources:
...
     name: hmcts/cnp-azuredevops-libraries
     endpoint: 'hmcts (1)'
 
+#variables:
+#  terraformVersion: 0.14.6
+#  agentPool: 'ubuntu-18.04'
```
Comment (timja) [inline azure-pipelines.yml:20]: remove commented out code

**Review by timja — CHANGES_REQUESTED** [review]
Can you check why there's a diff happening on demo? there should be no diff as far as I can tell, master branch failing? or need running?

File: azure-pipelines.yml (hunk 1 above)
Comment (madjava) [reply azure-pipelines.yml:14]: Yeah, would be reverted before merged, was just a test

Comment (madjava) [reply azure-pipelines.yml:14]: @taherkhan30 ok to revert this now and do another review. `cnp` merged.

File: azure-pipelines.yml (hunk 2 above)
Comment (madjava) [reply azure-pipelines.yml:20]: @taherkhan30 ^^

**Review by timja — COMMENTED** [review]
can you check the diff please on the record?

**Comment by taherkhan30** [comment]
> 
the code has an updated ip address for toffee A record [here](https://github.com/hmcts/azure-public-dns/blob/e610e46bd76cac5bf60c88c1c3400d5613bdaad9/environments/demo/demo-platform-hmcts-net.yml#L9)

in the portal the toffee A record is 51.142.81.239

**Comment by timja** [comment]
Can you make sure it’s fixed separate to this PR please?

**Review by timja — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ addition or removal was made.
 ## Limitations
 
 This module supports only the following DNS record types, due to limitations
-of the underlying Terraform provider:
+of the underlying Terraform provider::
```
Comment (timja) [inline README.md:83]:
```suggestion
of the underlying Terraform provider:
```

Comment (taherkhan30) [reply README.md:83]: i merged it, but ill change this.

**Pull request merged**